### ISSUE TITLE
Rework template variables

### ISSUE THREAD

**Issue by kblomqvist** [kblomqvist/yasha#11]
Consider the following variable (yaml) and template files:

```yml
---
foo: 1
bar: 2
baz: 3
```

```jinja2
foo * bar + baz = {{ foo * bar + baz }}
```

Wouldn't it be nice to be able to override the variables from command-line call. For example, the following command-line call introduces the inline variables and will return `foo * bar + baz = 9`:

```bash
yasha -V defaults.yml -v foo 2 -v bar 3 template.j2
```

This change would change the behavior of `-v --variables` option. The capital `-V` would be now used for variable files and -v for inline variables.

```python
@click.option("-V", type=click.File("rb"), multiple=True)
@click.option("-v", type=(str, str), multiple=True)
```

**Comment by kblomqvist** [comment]
Making `-V` to support multiple options (variable files), it makes no sense to have automatic file look up anymore? Similarly `-E` could allow giving multiple extension files.

**Comment by kblomqvist** [comment]
Solved in 7877475a71016a352064964b6cf98b5525fd35fa.
